getpaid/models.py

```diff
@@ -123,6 +123,9 @@ def on_failure(self):
         """
         self.change_status('failed')
 
+    def backend_display(self):
+        return self.backend.split(".")[-1].capitalize()
```

Comment:
I think this belongs to a presentation layer and I don't think you use this anywhere in `getpaid` so I think it's better to leave it out. Does this need to be included for your backend to work? You can always add similar util to your own project code right?